test/builtin/calculus/test_solve.py

```diff
@@ -8,6 +8,10 @@
 
 def test_solve():
     for str_expr, str_expected, message in (
+        # Clean the definitions, because
```

Comment:
@rocky, I pushed this directly here by mistake. In any case, this seems to make all the tests pass now.

Reply:
Thanks! You should not feel that doing this is a mistake! In the future if you want to push directly to the branch, please do.